doc/caveats.md

```diff
@@ -0,0 +1,41 @@
+Caveats
+=======
+
+Git-dit is, obviously, git specific and will hence not work with other VCS like
+mercurial. Import and export functionality for other issue tracking systems are,
```

Comment:
My annotation here was probably lost due to the rebase: Do not mention a VCS by name here. Just "others" is sufficient.